simulator/src/css/main.stylesheet.css

```diff
@@ -313,7 +313,7 @@ input[type="text"]:focus {
     text-align: center;
     position: absolute;
     left: 50%;
-    transform: translate(-50%, 13px);
+    transform: translate(-55%, 13px);
```

Comment:
This changes causing issues in other parts. Check the image, the menus are misalligning. Little bit inclined to  right.

![image](https://github.com/CircuitVerse/CircuitVerse/assets/57363826/9da59d6f-54fa-4fc4-9e63-69bc589cb3d4)

Reply:
I have tried it to 51% to 54% but it is not working for admin drop-down, Let's collaborate to find an alternative solution!
[asdfg.webm](https://github.com/CircuitVerse/CircuitVerse/assets/156365622/2c3c48b4-8ac8-488c-8d15-1c59fdd5a8d2)

Reply:
hello @tanmoysrt bhaiya, here one option we cango for is creating a separate class for the admin's dropdown and then assigning it a 55%, even with this approach, the issue with inclination for the admin's dropdown remains unresolved. And if we reduce the dropdown size or font size results in an odd appearance.

Reply:
It's not a option to change from 50% to some arbitary value. Because this thing is unreliable based on screen size. Find the actual issue the dropdown can't overflow if the outer div don't allow that. Try to check that.

For my screen size, the issue is not re-producable, Can you please check once on your side again (on the prod website) ?

Reply:
For small name the issue is appearing , for large name the issue is not reproducable